### DISCUSSION TITLE
Empty F-PLAN page in MCDU

### DISCUSSION THREAD

**Discussion by NachoTemino** [flybywiresim/aircraft#7373] [Help]
Hi, Have an issue with a empty flightplan screen in the MCDU, as in this picture:

![169266112-8e753d54-a9da-4f6d-9dba-f9496d068b58](https://user-images.githubusercontent.com/110159030/181481326-226b6af5-2d30-4dd5-bc37-49e412425bac.jpeg)

I have installed, deinstalled A320nx and tried every version (stable, dev, experimental..). I have unistalled a reinstalled MSFS. I have even Reinstalled Windows 11.

After a fresh Windows 11 install, a fresh MSFS install with no other addon but the A320nx I Use the "usual" procedure:
- Dispatch a FP in Simbrief
- Start the flight only with the departure airport, at a gate not a the runway.
- Set Simbrief user in the flypad
- Load the flight from simbrief in the flightpad. Set Weights and fuel.
- From Cold and dark airplane I set power to the systems both external power and batteries.
- Align IRS system
- MCDU ATSU -> AOC -> INIT Data request
- MCDU ATSU -> AOC -> W/B -> Request Send -> Boarding Start
- From the INIT-A page -> Init request
- Everything looks ok, departure and arrival airports apperas in the INIT page
But then I press F-PLN key in the MCDU keyboard and that same blank page with the strange squares at the top appears

What the hell am I doing wrong? FWB A320nx is the only reason I use MSFS.
I am thinking of going back to P3D if I cannot find a soultion for this.

**Comment by tracernz** [comment]
FBW only supports the latest version of MSFS...
